### PR TITLE
Search event by keyword

### PR DESCRIPTION
**Pull request by mertzjl91**
Searches events by keyword.

### CODE REVIEW

File: src/main/java/com/launchcode/AroundTownServer/models/Event.java [hunk 1]
```diff
@@ -11,11 +11,11 @@
 @Entity
 public class Event {
 
-    private String name;
+    public String name;
```
Comment (miketully426) [inline src/main/java/com/launchcode/AroundTownServer/models/Event.java]: This should all be private.

File: src/main/java/com/launchcode/AroundTownServer/models/Event.java [hunk 2]
```diff
@@ -11,11 +11,11 @@
...
 
-    private String description;
+    public String description;
```
Comment (miketully426) [inline src/main/java/com/launchcode/AroundTownServer/models/Event.java]: private!

File: src/main/java/com/launchcode/AroundTownServer/models/Event.java [hunk 3]
```diff
@@ -11,11 +11,11 @@
...
 
-    private String location;
+    public String location;
```
Comment (miketully426) [inline src/main/java/com/launchcode/AroundTownServer/models/Event.java]: private!

**Review by miketully426 — CHANGES_REQUESTED** [review]

**Comment by miketully426** [comment]
I think you forgot to push some commits? There's no code here for searching.

**Review by lfallert — APPROVED** [review]
Tested and working as intended. I did update a couple event fields to have the correct access modifiers and I pushed those changes to this branch.

**Review by CNichole726 — APPROVED** [review]
Looks good to me

**Pull request closed without merging**